src/settings-ui/Settings.UI/ViewModels/GeneralViewModel.cs

```diff
@@ -298,12 +301,13 @@ public bool EnableWarningsElevatedApps
             }
         }
 
-        // Are we running a dev build? (Please note that we verify this in the code that gets the newest version from GitHub too.)
-        public static bool AutoUpdatesDisabledOnDevBuild
+        public bool SomeUpdateSettingsAreGpoManaged
         {
             get
             {
-                return Helper.GetProductVersion() == "v0.0.1";
+                return _newUpdatesToastIsGpoDisabled ||
+                    (_isAdmin && _autoDownloadUpdatesIsGpoDisabled) ||
```

Comment:
Hi @htcfreek  Why the `_isAdmin` check in here?

Reply:
Same as before.